client/src/App.js

```diff
@@ -1,32 +1,14 @@
 import React, { Component } from 'react';
```

Comment:
I would reorder your imports. Group the css with each other and group the components with each other. Same comment applies in the other components.